packages/airnode-adapter/src/response-processing/extraction.test.ts

```diff
@@ -1,4 +1,5 @@
 import { ethers } from 'ethers';
+import { MAX_ENCODED_RESPONSE_SIZE } from 'src/constants';
```

Comment:
```suggestion
import { MAX_ENCODED_RESPONSE_SIZE } from '../constants';
```

Reply:
My VS code does this a lot - does it happen to you as well?

Reply:
*ahem* I use Neovim

Reply:
based